server/config/config.go

```diff
@@ -376,7 +382,27 @@ type OrgConfig struct {
 	Domain string `yaml:"domain" usage:"Your organization's email domain. If this is set, only users with email addresses in this domain will be able to register for a BuildBuddy account."`
 }
 
-var sharedGeneralConfig generalConfig
+var (
+	sharedGeneralConfig  = &generalConfig{}
+	originalStringSlices = make(map[string][]string)
```

Comment:
Can you add a short comment describing what originalStringSlices and originalSetFlags contain?